cmake/presets/compilers/clang.json

```diff
@@ -7,6 +7,9 @@
       "environment": {
         "CC": "clang",
         "CXX": "clang++"
+      },
+      "cacheVariables": {
+        "CMAKE_CXX_FLAGS": "-stdlib=libc++"
```

Comment:
This was needed to fix broken CI builds for clang on Ubuntu. Not strictly related to this PR.